PendingReleaseNotes.md

```diff
@@ -17,6 +17,7 @@
     fix this to be `<dataDirHostPath>/log/<namespace>`, the same as other daemons.
   - Use the mon configuration database for directory-based OSDs, and do not generate a config
 - A new ceph-crashcollector controller has been added, that new pod will run on any node where a Ceph pod is running. Read more about this in the [doc](Documentation/ceph-cluster-crd.html#cluster-wide-resources-configuration-settings)
+- The Ceph version detection job now has its `app` label applied correctly
```

Comment:
Sounds good to backport. In that case, let's not update the pending release notes since we know it will cause a conflict.

Reply:
Done.